tt_metal/hw/firmware/src/ncrisc.cc

```diff
@@ -95,8 +95,14 @@ int main(int argc, char *argv[]) {
 
         uint32_t kernel_config_base = firmware_config_init(mailboxes, ProgrammableCoreType::TENSIX, DISPATCH_CLASS_TENSIX_DM1);
         uint32_t tt_l1_ptr *cb_l1_base = (uint32_t tt_l1_ptr *)(kernel_config_base +
-            launch_msg->kernel_config.cb_offset);
-        setup_cb_read_write_interfaces(cb_l1_base, 0, launch_msg->kernel_config.max_cb_index, true, true, false);
+            launch_msg->kernel_config.local_cb_offset);
```

Comment:
threshold doesn't mean much to me - threshold to what?  I think max may be better?
this sequence shows up a few places, wonder if it should move to firmware_common code

Reply:
I named it threshold since I reused the variable for the local and remote cb setup, so it's contains the max local cb index, then the min remote cb index. I can rename/separate them